Reproducibility: how to use rustc wrapper (#1489)
casper-network/docs: source/docs/casper/developers/writing-onchain-code/getting-started.md

If you look inside the newly-created _my-project_ folder, you will find two crates: `contract` and `tests`. This is a complete basic smart contract that saves a value, passed as an argument, on the blockchain. The `tests` crate provides a runtime environment of the Casper virtual machine, and a basic smart contract test.

### Reproducibility

> Currently, [cargo](https://github.com/rust-lang/cargo/issues/8140) does not provide cross-platform reproducibility for binary files, including WebAssembly. The ability to compile a smart contract to exactly the same binary file is important, for example, when there is a need to verify the smart contract binary stored on the blockchain is exactly the same as the provided source code.

Reviewer: ```suggestion
Currently, [cargo](https://github.com/rust-lang/cargo/issues/8140) does not provide cross-platform reproducibility for binary files, including WebAssembly. The ability to compile a smart contract to the same binary file is important, for example, when verifying that the smart contract binary stored on the blockchain is the same as the provided source code.
```
